### ISSUE TITLE
Debugging with Webpack doesn't work

### ISSUE THREAD

**Issue by TechStudent10** [microsoft/vscode-js-debug#1527]
### TL;DR
When using the debugger in an Electron environment that takes advantage of React in the renderer process, I cannot set breakpoints in the TypeScript files.

---

I am using an Electron + (React + TypeScript) combo (Electron = JavaScript, React = TypeScript that gets bundled via Webpack into JavaScript). Over the past couple of days, I've been trying to get VSCode's debugger to stop on a breakpoint in the renderer process. 

The debugger works fine with the main process, but not with the renderer process.

I think this is the case due to the fact that I am using bundled code, so breakpoints set in the original source code won't transfer. This may also be due to the fact that this is an Electron application. I've tried the "Debugger for Chrome" extension and it's newer version, but they also do not work.

Any help would be appreciated.

**Comment by connor4312** [comment]
Please collect logs using the instructions in the "bug" issue template

**Comment by TechStudent10** [comment]
> Please collect logs using the instructions in the "bug" issue template

@connor4312 `trace: true` does nothing. I see nothing the debug console.

**Comment by connor4312** [comment]
Please collect a recording of what you see. https://gifcap.dev/ is useful

**Comment by TechStudent10** [comment]
I just realized that I did somethings wrong:

- I had to enable the `sourceMap` devtool in webpack
- I used this guide and the `launch.json` configs provided: https://vscode-debug-specs.github.io/javascript_electron/
